File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.js
function solution(s, skip, index) {
    let answer = '';
    const atoz = 'abcdefghijklmnopqrstuvwxyz'.match(
        new RegExp(`[^${skip}]`, 'g'),
    );
    for (const w of s) {
        const newIdx = atoz.indexOf(w) + index;
        answer += atoz[newIdx % atoz.length];
    }
  return answer;
}

File: 프로그래머스/1/155652. 둘만의 암호/둘만의 암호.py
def solution(s, skip, index):
    answer = ""
    atoz = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm',
            'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']

    for v in skip:
        atoz.remove(v)

    for w in s:
        answer += atoz[(atoz.index(w)+index)%len(atoz)]

    return answer